### ISSUE TITLE
db tables full - no stations can start

### ISSUE THREAD

**Issue by marekdziem** [AzuraCast/AzuraCast#6106]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

all the stations have stopped!
Initial error shown:
![image](https://user-images.githubusercontent.com/3094096/220549718-4b0e3e73-f37a-4371-bab5-38085ef10c75.png)

When trying to start stations:
SupervisorException at /var/azuracast/www/src/Exception/SupervisorException.php L69: station_8:station_8_frontend encountered an error: SpawnErrorException
Check the log for details.

mariadb log shows tables are full!

2023-02-21 12:18:42 71220 [ERROR] InnoDB: preallocating 322961408 bytes for file ./azuracast/listener.ibd failed with error 28
2023-02-21 12:18:42 71220 [ERROR] mysqld: The table 'listener' is full

how to fix? 
URGENT!!
@BusterNeece

**Comment by Vaalyn** [comment]
Check if your servers HDD is full via `df -h`

The database is located in `/var/lib/docker/volumes` so you'll need to check if the partition where that path is located at is full.

If that partition is full you'll need to increase the size of the partition of your server or delete files from it that you don't need anymore.

**Comment by marekdziem** [comment]
Hey thanks so much for the pointers Vaalyn!

I freed up some files – and now to figure out how to size up the 60gb partition!

 �

**Comment by sachinshakya507** [comment]
Hey @marekdziem what are the files you deleted in order to free up space?

**Comment by marekdziem** [comment]
Hi Sachin,

 �

The main culprit happened to be old lingering full media backups not immediately visible to azuracast admin on the backup pages!

 �

I began troubleshooting by looking at azuracast logs, initially checking MariaDb after seeing log entries saying tables are full and can’t be written to. I found the listener table had grown to approx. 290Mb! That kept me barking up the wrong tree for a while!

Thank for the support guys!

Love

marek

 �

 �

 �
